doc/source/dev/development_workflow.rst

```diff
@@ -150,6 +153,81 @@ been added to ``upstream`` that affect your work. In this case, follow the
 :ref:`rebasing-on-main` section of this document to apply those changes to
 your branch.
 
+.. _formatting-with-pre-commit:
+
+Automatic code formatting with pre-commit
+-----------------------------------------
+
+NymPy recommends that code follow pep7/pep8 guidelines, but does not currently
```

Comment:
typo in first word here

also `pep7` -> PEP 7 (and same for 8)

Reply:
Got it in b1b10faa0